independent-publisher-connectors/XSOAR/Readme.md

```diff
@@ -0,0 +1,23 @@
+## XSOAR Connector
+XSOAR provides a security orchestration, automation, and remediation platform.  Using this integration you can submit data to XSOAR for usage in security automations.
+
+## Publisher: Landon Chelf | FIS
+
+## Prerequisites
+You will need the following to proceed:
+* A XSOAR with an integration instance available over HTTPS via [TODO HERE].
+* An API key for the integration instance
+
+## Supported Operations
+The connector supports the following operation:
+* `Send to XSOAR`: Send JSON data to XSOAR
+
+## Obtaining Credentials
+API keys are created by configuring an integration instance within XSOAR.
```

Comment:
@landonc Can you please provide more details in step-by-step format to obtain an API key from XSOAR